WordPress/Classes/Services/PostRepository.swift

```diff
@@ -390,7 +390,7 @@ extension PostRepository {
     ///   - statuses: Only fetch pages whose status is included in the given statues.
     ///   - blogID: Object ID of the site.
     /// - Returns: A `Task` instance representing the fetching. The fetch pages API requests will stop if the task is cancelled.
-    func fetchAllPages(statuses: [BasePost.Status], in blogID: TaggedManagedObjectID<Blog>) -> Task<[TaggedManagedObjectID<Page>], Swift.Error> {
+    func fetchAllPages(statuses: [BasePost.Status], authorUserID: NSNumber? = nil, in blogID: TaggedManagedObjectID<Blog>) -> Task<[TaggedManagedObjectID<Page>], Swift.Error> {
```

Comment:
Nitpick: The Swift API Guidelines [recommend to locate parameters with default values at the end of the list](https://swift.org/documentation/api-design-guidelines/#parameter-with-defaults-towards-the-end):

> **Prefer to locate parameters with defaults toward the end of the parameter list.** Parameters without defaults are usually more essential to the semantics of a method, and provide a stable initial pattern of use where methods are invoked.

Reply:
Good catch. I used default parameter value quite freely in a few similar functions in this class. I want to keep the blog parameter as the last parameter (or the first), and rest of the parameters (which act as post/pages filters) don't really have an sensible order. But these functions are added one by one across many PRs, it's probably worth looking into these function holistically later.

Reply:
Sounds good